template/src/pageComponents/buy-me-coffee/ContractDemo.tsx

```diff
@@ -27,16 +58,10 @@ export default function BuyMeCoffeeContractDemo() {
         <h2 className="mb-5 w-fit text-2xl font-semibold text-white">Messages from supporters</h2>
         {memos?.length > 0 && <Memos memos={memos} />}
       </section>
-      <aside
-        className={clsx([
-          'rounded-3xl border border-solid border-boat-color-palette-line',
-          'mt-10 bg-boat-color-palette-backgroundalternate p-10 md:mt-0',
-        ])}
-      >
-        <h2 className="mb-5 w-full text-center text-2xl font-semibold text-white lg:text-left">
-          Buy Me a Coffee!
-        </h2>
-        <FormBuyCoffee onComplete={handleOncomplete} />
+      <aside>
+        <div className="mt-10 rounded-3xl border border-solid border-boat-color-palette-line bg-boat-color-palette-backgroundalternate p-10 md:mt-0">
```

Comment:
Let's use `clx` for long tailwind cases so we can split in multiple lines.

Reply:
Done :)